src/Elevators/Components/Pages/Home.razor

```diff
@@ -2,67 +2,24 @@
 
```

Comment:
sem hodit
`@Page "/"` a klidně i pod to `@Page "/home"`
aby byla stránka něco jako index.html